kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -55,6 +55,16 @@ def __get_authentication_config(self):
             "sasl.kerberos.service.name": self.config._sasl_kerberos_service_name,
         }
 
+        if self.config._sasl_mechanism == "OAUTHBEARER":
+            extras_parameters['sasl.oauthbearer.method'] = "oidc"
+            extras_parameters["sasl.oauthbearer.client.id"] = (self.config._sasl_oauth_token_provider.get("client_id"),)
+            extras_parameters["sasl.oauthbearer.token.endpoint.url"] = (
+                self.config._sasl_oauth_token_provider.get("url"),
+            )
+            extras_parameters["sasl.oauthbearer.client.secret"] = (
+                self.config._sasl_oauth_token_provider.get("client_secret"),
+            )
```

Comment:
Aren't you creating tuples here? 

```suggestion
            extras_parameters["sasl.oauthbearer.client.id"] = self.config._sasl_oauth_token_provider.get("client_id")
            extras_parameters["sasl.oauthbearer.token.endpoint.url"] = (
                self.config._sasl_oauth_token_provider.get("url")
            )
            extras_parameters["sasl.oauthbearer.client.secret"] = (
                self.config._sasl_oauth_token_provider.get("client_secret")
            )
```

Reply:
Fixed!